torch/csrc/jit/operator_upgraders/upgraders_entry.cpp

```diff
@@ -57,19 +58,39 @@ def full_out_0_4(size:List[int], fill_value:number, *, out:Tensor) -> Tensor:
   return torch.full(size, fill_value, out=out)
 )SCRIPT"}});
 
+std::shared_ptr<Graph> create_upgrader_graph(
+    const std::string& upgrader_name,
+    const std::string& upgrader_body) {
+  auto cu = std::make_shared<CompilationUnit>();
+  cu->define(c10::nullopt, upgrader_body, nativeResolver(), nullptr);
+  Function& jitFunc = cu->get_function(upgrader_name);
+  GraphFunction& graphFunction = toGraphFunction(jitFunc);
+  return graphFunction.graph();
+}
 using UpgraderMap = std::unordered_map<std::string, std::shared_ptr<Graph>>;
 void populate_upgraders_graph_map() {
   UpgraderMap populate_content;
   for (const auto& entry : kUpgradersEntryMap) {
-    auto cu = std::make_shared<CompilationUnit>();
-    cu->define(c10::nullopt, entry.second, nativeResolver(), nullptr);
-    Function& jitFunc = cu->get_function(entry.first);
-    GraphFunction& graphFunction = toGraphFunction(jitFunc);
-    populate_content.insert(std::make_pair(entry.first, graphFunction.graph()));
+    auto upgrader_graph = create_upgrader_graph(entry.first, entry.second);
+    populate_content.insert(std::make_pair(entry.first, upgrader_graph));
   }
-
   populate_upgraders_map(std::forward<UpgraderMap>(populate_content));
```

Comment:
Why is the `forward` here necessary? It should only be needed for arguments or rvalues. here `populate_content` is neither. What do you think?